src/CodeViewer/Viewer.cpp

```diff
@@ -488,7 +505,11 @@ void Viewer::SetAnnotatingContent(absl::Span<const AnnotatingLine> annotating_li
     }
   }
 
+  largest_occuring_line_number_main_content_ = blockCount();
```

Comment:
Following the comment I wrote before, this doesn't count the empty blocks or they are joined automatically together?